src/commands/start.js

```diff
@@ -22,11 +28,25 @@ class StartCommand extends BaseCommand {
       update: isUpdate,
       'drive-image-build-path': driveImageBuildPath,
       'dapi-image-build-path': dapiImageBuildPath,
+      'mine-blocks': mineBlocks,
     },
     dockerCompose,
     startNodeTask,
     config,
   ) {
+    let blockTimeMs;
+
+    if (mineBlocks !== null) {
```

Comment:
Let's add new options to config that could be overwritten by command options.:
 - `core.miner.enabled=false`
 - `core.miner.interval="2.5m"`
 - `core.miner.address=null`

In this case, you don't need to pass them every time.